fspiop/v2_0_ISO20022/openapi3/components/schemas/BaseOneRate.yaml

```diff
@@ -2,5 +2,4 @@ title: BaseOneRate
 type: string
 description: |
   Rate expressed as a decimal, for example, 0.7 is 7/10 and 70%.
-pattern: ^(?=[\d\-\.\,]{1,11}$)([\d\-\,\.]{1,10})?$
```

Comment:
If the json schema for this is:
  <xs:simpleType name="BaseOneRate">
        <xs:restriction base="xs:decimal">
            <xs:fractionDigits value="10"/>
            <xs:totalDigits value="11"/>
        </xs:restriction>
    </xs:simpleType>
The this regex is all the possible options.    
pattern: ^(?!0\d)(\d{1,11}|\d{1,10}\.\d{1}|\d{1,9}\.\d{1,2}|\d{1,8}\.\d{1,3}|\d{1,7}\.\d{1,4}|\d{1,6}\.\d{1,5}|\d{1,5}\.\d{1,6}|\d{1,4}\.\d{1,7}|\d{1,3}\.\d{1,8}|\d{1,2}\.\d{1,9}|\d{1}\.\d{1,10})$

That seems a bit excessive. Maybe this is ok. I.e. is only support max 6 orders of magnitiude difference is currencies. 
pattern: ^(?!0\d)(\d{1,6}|\d{1,5}\.\d{1,5}|\d{1,4}\.\d{1,6}|\d{1,3}\.\d{1,7}|\d{1,2}\.\d{1,8}|\d{1}\.\d{1,9})$